toonz/sources/toonz/filebrowsermodel.cpp

```diff
@@ -1165,22 +1152,19 @@ void DvDirModelRootNode::refreshChildren() {
     DvDirModelSpecialFileFolderNode *child;
     child = new DvDirModelSpecialFileFolderNode(this, L"My Documents",
                                                 getMyDocumentsPath());
-    child->setPixmap(recolorPixmap(svgToPixmap(
-        getIconThemePath("actions/16/my_documents.svg"), QSize(16, 16))));
+    child->setPixmap(generateIconPixmap("my_documents"));
     m_specialNodes.push_back(child);
     addChild(child);
 
     child =
         new DvDirModelSpecialFileFolderNode(this, L"Desktop", getDesktopPath());
-    child->setPixmap(recolorPixmap(svgToPixmap(
-        getIconThemePath("actions/16/desktop.svg"), QSize(16, 16))));
```

Comment:
@konero To fix the oversized icons that were showing up in the File browser dialog, I had to introduce the QSize(16,16) on a few of the icons.  See commit https://github.com/tahoma2d/tahoma2d/commit/cea78e587a7e420485e8b6e2f75bd6fd8780d126

Removing this is causing these icons to appear large again.  Not sure if the image itself needs a change or not.

Reply:
Thanks, I resolve this in latest commit differently